src/utils/clmfates_interfaceMod.F90

```diff
@@ -820,7 +842,23 @@ subroutine dynamics_driv(this, nc, bounds_clump,      &
                   atm2lnd_inst%wind24_patch(p)
 
          end do
-         
+
+         ! Here we use the same logic as the pft_areafrac initialization to get an array with values for each pft
+         ! in FATES. 
+         ! N.B. Fow now these are fixed values pending HLM updates. 
+         if(use_fates_sp)then
```

Comment:
Get "tlai" and "tsai" from CLM only in "dynamics_driv"  is not enought.  When model is initialized, this part of the code will not be called. As a results, "tlai" and "tsai" will be initialized with arbitrary values. This will lead to balance error at a later step.

Reply:
More details about the initialization issue can be checked at the following link: https://github.com/huitang-earth/fates-sp_initialization_issue

Reply:
has this issue been fixed now?

Reply:
Fixed per https://github.com/ESCOMP/CTSM/pull/1182/commits/97a0ad8e87536e9db7508691af3fb02c6c4d1f46